api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -11,6 +11,101 @@
            http://www.springframework.org/schema/util
            http://www.springframework.org/schema/util/spring-util-2.0.xsd">
 
+
+	<bean id="extendedPatientService" class="org.springframework.transaction.interceptor.TransactionProxyFactoryBean">
```

Comment:
Any reason you need this to be it's own service?  Couldn't you just add these methods into the YendaNafePatientRegistrationServiceImpl and not need multiple services?

Reply:
@mseaton The thinking behind the extendedPatientService is to extend the existing PatientService in the core. The patientService doesn't have a method to get a patient Identifier by identifier type (e.g NNO xxx) so i implemented the method separately to handle that. Implementing separately was chosen instead of implementing in YendaNafePatientRegistrationServiceImpl because the function is generic and can get identifiers even those not in YendaNafe(e.g OpenMRS Identification Number, ARV Number, IC3 Identifier). So if there is a need someday to get a patient identifier by identifier type, this service is available rather than coupling the function in a class with a specific responsibility.